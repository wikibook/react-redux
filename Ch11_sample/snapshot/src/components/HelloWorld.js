import React from 'react';

const HelloWorld = () => <div>Hello React!</div>;

export default HelloWorld;