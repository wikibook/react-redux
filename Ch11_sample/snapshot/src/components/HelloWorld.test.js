import React from 'react';
import renderer from 'react-test-renderer';
import HelloWorld from './HelloWorld';

test('HelloWorld 컴포넌트 스냅숏 테스트', () => {
  const result = renderer.create(<HelloWorld/>).toJSON();

  expect(result).toMatchSnapshot();
});