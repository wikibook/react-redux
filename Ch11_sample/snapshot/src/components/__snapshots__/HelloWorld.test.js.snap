// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld 컴포넌트 스냅숏 테스트 1`] = `
<div>
  Hello React!
</div>
`;
